composer.json

```diff
@@ -21,7 +21,7 @@
         "illuminate/validation": "^9.21"
     },
     "conflict": {
-        "laravel/framework": "<9.19.0"
+        "laravel/framework": "<9.37.0"
```

Comment:
Taking a guess that this is the version we release this in. Will need to confirm on release day.